inet/vibe/inet/webform.d

```diff
@@ -220,24 +247,56 @@ struct FilePart {
 private bool parseMultipartFormPart(InputStream)(InputStream stream, ref FormFields form, ref FilePartFormFields files, const(ubyte)[] boundary, size_t max_line_length)
 	if (isInputStream!InputStream)
 {
+	//find end of quoted string
+	auto indexOfQuote(string str) {
+		ptrdiff_t res;
+
+		foreach (ch; str) {
+			if (ch == '"' && (res == 0 || str[res-1] != '\\')) return res;
+			res++;
+		}
+
+		return -1;
```

Comment:
Probably would be the best idea to throw an exception here. Otherwise this would result in an out-of-bounds error in `parseValue`, which terminates the application.